invoice/invoice.go

```diff
@@ -15,15 +15,19 @@ import (
 func CreateInvoice(ctx context.Context, client *model.Odoo, invoice invoice.Invoice, options ...Option) (int, error) {
 	opts := buildOptions(options)
 
-	name := fmt.Sprintf("APPUiO Cloud %s %d", invoice.PeriodStart.Month(), invoice.PeriodStart.Year())
-	toCreate := opts.invoiceDefaults
-	toCreate.Name = name
-	toCreate.Date = odoo.Date(opts.InvoiceDateOrNow())
-
 	partnerID, err := strconv.Atoi(invoice.Tenant.Target)
 	if err != nil {
 		return 0, fmt.Errorf("error converting tenant target to int: %w", err)
 	}
+	partner, err := client.FetchPartnerByID(ctx, partnerID)
+	if err != nil {
+		return 0, fmt.Errorf("error fetching partner info from Odoo: %w", err)
+	}
```

Comment:
not sure if it's correct to do this here...

Reply:
it's ok